apps/next/src/popup/app.tsx

```diff
@@ -153,6 +154,7 @@ export function App() {
           <DefiContextProvider />,
           <FirebaseContextProvider />,
           <NextUnifiedBridgeProvider />,
+          <WalletTotalBalanceProvider />,
```

Comment:
Now that I think about it, maybe it's not the best place. This way any approval screen, even something simple like `hey, let's switch to Ethereum`, will trigger the balance fetching for all of the wallets.

I think we should add it in `AppRoutes.tsx` instead, but leave it out of `ApprovalRoutes.tsx` - this way it will trigger balance refresh when user opens up the extension UI, but will not be triggered on some approval screens that don't need the freshest balances.